### ISSUE TITLE
SGD Classifier: sklearn.linear_model._sgd_fast.Hinge

### ISSUE THREAD

**Issue by AHReccese** [openscilab/pymilo#73]
#### Description
fix loss function serialization issue in SGDClassifier in scikit1.4.0
